scalafix-testkit/src/main/scala/scalafix/testkit/SyntacticRuleSuite.scala

```diff
@@ -8,16 +8,20 @@ import scalafix.syntax._
 
 import org.scalatest.FunSuiteLike
 
-class SyntacticRuleSuite(rule: Rule) extends FunSuiteLike with DiffAssertions {
-  def check(name: String, original: String, expected: String): Unit = {
+class SyntacticRuleSuite extends FunSuiteLike with DiffAssertions {
+  def check(
+      rule: Rule,
+      name: String,
+      original: String,
+      expected: String): Unit = {
     test(name) {
       import scala.meta._
       val obtained = rule.apply(Input.String(original))
       assertNoDiff(obtained, expected)
     }
   }
 
-  def checkDiff(original: Input, expected: String): Unit = {
```

Comment:
Forward this to new `checkDiff` overload with `defaultRule`

Reply:
Done.